clickhouse-client/src/test/java/com/clickhouse/client/ClickHouseServerForTest.java

```diff
@@ -289,5 +379,44 @@ public static void afterSuite() {
         if (clickhouseContainer != null) {
             clickhouseContainer.stop();
         }
+
+        if (isCloud()) {
+            ClickHouseNode server = ClickHouseNode.builder(ClickHouseNode.builder().addOption(ClickHouseClientOption.SSL.getKey(), "true").build())
+                    .address(ClickHouseProtocol.HTTP, new InetSocketAddress(System.getenv("CLICKHOUSE_CLOUD_HOST"), 8443))
+                    .credentials(new ClickHouseCredentials("default", getPassword()))
+                    .options(Collections.singletonMap(ClickHouseClientOption.SSL.getKey(), "true"))
+                    .build();
+
+            //Remove database after testing
+            boolean success = false;
+            int retries = 0;
```

Comment:
Could we extract retry logic outside? reading inlined code that mixes responsibilities is hard